app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -94,6 +97,14 @@ abstract class RoutePresenter(
         alertDialogStore.showDialog(activity, destination)
     }
 
+    fun showToastNotification(action: ToastNotificationAction) {
+        try {
+            (activity as RootActivity).showToastNotification(action)
```

Comment:
You should inline this so the `RootPresenter`. 

It might help to start with 

```kotlin
activity.apply {
    // `this` is now the activity.
}
```

Reply:
ctrl T + inline